src/Services/Warehouse/Query/Application/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using Application.Abstractions;
+using Application.UseCases.Events;
+using Application.UseCases.Queries;
+using Contracts.Abstractions.Paging;
+using Contracts.Services.Warehouse;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Application.DependencyInjection;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddEventInteractors(this IServiceCollection services)
+        => services
+            .AddScoped<IProjectInventoryDetailsWhenChangedInteractor, ProjectInventoryDetailsWhenChangedInteractor>()
+            .AddScoped<IProjectInventoryItemWhenChangedInteractor, ProjectInventoryItemWhenChangedInteractor>();
```

Comment:
Interactor names must match with the projections and protobuf

Reply:
ok, I'll refactor it.